src/sentry/static/sentry/app/components/events/interfaces/threads/threadSelector/option.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {css} from '@emotion/core';
+
+import {Color} from 'app/utils/theme';
+import {t, tct} from 'app/locale';
+import Tooltip from 'app/components/tooltip';
+import TextOverflow from 'app/components/textOverflow';
+import {EntryTypeData} from 'app/types';
+import {IconFire, IconCheckmark} from 'app/icons';
+
+import {Grid, GridCell} from './styles';
+
+type Props = {
+  id: string;
+  details: ThreadInfo;
+  crashedInfo?: EntryTypeData;
+  name?: string | null;
+  crashed?: boolean;
+};
+
+type ThreadInfo = {
+  label?: string;
+  filename?: string;
+};
+
+const Option = ({id, details, name, crashed, crashedInfo}: Props) => {
+  const {label = t('<unknown>'), filename = t('<unknown>')} = details;
+  const optionName = name || t('<unknown>');
+  return (
+    <Grid>
+      <GridCell>
+        <InnerCell>
+          <Tooltip title={`#${id}`} position="top">
+            <TextOverflow>{`#${id}`}</TextOverflow>
+          </Tooltip>
+        </InnerCell>
+      </GridCell>
+      <GridCell>
+        <InnerCell isBold>
+          <Tooltip title={optionName} position="top">
+            <TextOverflow>{optionName}</TextOverflow>
+          </Tooltip>
+        </InnerCell>
+      </GridCell>
+      <GridCell>
+        <InnerCell color="blue">
+          <Tooltip title={label} position="top">
+            <TextOverflow>{label}</TextOverflow>
+          </Tooltip>
+        </InnerCell>
+      </GridCell>
+      <GridCell>
+        <InnerCell color="purple">
+          <Tooltip title={filename} position="top">
+            <TextOverflow>{filename}</TextOverflow>
+          </Tooltip>
+        </InnerCell>
+      </GridCell>
+      <GridCell>
+        <InnerCell isCentered>
+          {crashed ? (
+            crashedInfo ? (
+              <Tooltip
+                title={`${tct('errored with [crashedInfo]', {
+                  crashedInfo: crashedInfo.values[0].type,
+                })}`}
+                position="top"
+              >
+                <IconFire color="red" />
+              </Tooltip>
+            ) : (
+              <IconFire color="red" />
+            )
+          ) : (
+            <IconCheckmark color="green" size="xs" />
+          )}
+        </InnerCell>
+      </GridCell>
+    </Grid>
+  );
+};
+
+export {Option};
+
+const centerCss = css`
+  display: flex;
+  align-items: center;
+  height: 100%;
+`;
```

Comment:
Can we move this into the InnerCell css since it's not re-used?

Reply:
good catch!